worknenjoy/gitpay
# Sends email to task owner on assignment

## Related Issue

> #182

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user1
- Create Task
- Create /login to user2
- Click I'm interested from user2's account
- Assign the task to user2 from user1's account
- user1 should receive email as given in issue
